javascript/javascript1/week2/week2.js

```diff
@@ -0,0 +1,159 @@
+//fullname function
+function getFullname(fullname1, fullname2) {
+    return fullname1 + " " + fullname2
+}
+console.log(getFullname("Benjamin", "Hughes"))
+
+
+//Formal fullname
+function getFullname(fullname1, fullname2, useFormalName = false, gender = "male") {
+    if (useFormalName) {
+        if (gender === "male") {
+
+            return "Lord" + fullname1 + " " + fullname2
+        } else {
+            return "Miss" + fullname1 + " " + fullname2
+        }
+    } else {
+        return fullname1 + " " + fullname2
+    }
+
+}
+
+
+//Event application
+const day = ["sunday", "monday", "tuesday", "wendsday", "thursday", "friday", "saterday"]
+
+function getEventWeekday(today, eventDay) {
+    let eventDay_ = eventDay % 7;
+
+    let index = -1;
+    for (let i = 0; i < day.length; i++) {
+        if (today === day[i]) {
+            index = i;
+            break;
+        }
+    }
+
+    if (index === -1) {
+        return undefined;
+    }
+
+    return "The event will take place in " + day[(index + eventDay_) % 7];
+}
+
+console.log(getEventWeekday("tuesday", 41));
+
+
+
+
+
+// wear clothes
+function clothesToWear(temperature) {
+    if (temperature > 25) {
+        console.log("clothesToWear : " + "shorts and a t-shirt");
+    } else if (temperature > 10) {
+        console.log("clothesToWear : " + "Sweaters and coats");
+    } else
+        console.log("clothesToWear : " + "down jacket");
+}
+clothesToWear(15);
+
+
+
+//Student manager
+const class07Students = [];
+
+function addStudentToClass(studentName) {
+
+    if (studentName === "") {
+        console.log("You cannot add an empty string to a class");
+        return false;
+    }
+
+    for (let i = 0; i < class07Students.length; i++) {
+
+        if (studentName === class07Students[i]) {
+
+            console.log(class07Students[i] + " is already in the class");
+            return false;
+        }
+    }
+
+    if (studentName === "queen") {
+        class07Students.push(studentName);
+        console.log("queen is added to the class");
+        return true;
+    }
+
+    if (getNumberOfStudents() > 6) {
+        console.log("Cannot add more students to class 07");
+        return false;
+    }
+    else {
```

Comment:
you don't need else here.

Reply:
I think we need else, because there is 2 situation here, so can you check it again?

Reply:
if getNumberOfStudents() > 6 then it runs if-block and returns so it's not going to run a code after it. check this 
```
if (getNumberOfStudents() > 6) {
        console.log("Cannot add more students to class 07");
        return false;
    }
class07Students.push(studentName);
console.log(studentName + " is added to the class");
return true;
```